src/components/link/link.stories.mdx

```diff
@@ -184,6 +184,37 @@ perform client-side routing, but still provide the `href` to render an HTML link
   </Story>
 </Canvas>
 
+### On a dark background
```

Comment:
Should we also include the disabled link as well in this story, so it matches the figma design we received?